docs/pages/mintlist/[mintlistAddress].tsx

```diff
@@ -136,6 +151,27 @@ export default function MintlistPage() {
                 </div>
               </div>
             </div>
+            <div>
+              <h2>NFTs</h2>
+              {user &&
+                data.mintlist.authority === user.address &&
+                data.mintlist.mint_infos.length <
+                  data.mintlist.num_nfts_total && (
+                  <div className="mb-4">
+                    <p className="mb-2">
+                      NOTE: You can upload up to {MAX_NFTS_PER_BATCH} NFTs at
```

Comment:
This feels like an odd note both in UI and in restriction. I think we should allow uploading 10k NFTs at once and we should figure out how to split it into multiple transactions, otherwise this tool isn't that helpful to people who want to create large collections.

Reply:
yeah agree, I'll add this one to the list of polishes to deliver in next PR